docs/layouts/partials/menu/open-menu.html

```diff
@@ -1,30 +1,30 @@
 {{$currentNode := .}}
```

Comment:
This file is _copied_ from the theme and minimally modified to favor using each page's `titleLink` over its `title` when generating the menu. Hugo favors this file over the original in themes/...